roles/dnsmasq/defaults/main.yml

```diff
@@ -33,3 +33,6 @@ dnsmasq_min_replicas: 1
 
 # Custom name servers
 dnsmasq_upstream_dns_servers: []
+
+# Arch of Docker images and needed packages
+image_arch: "{{ image_arch | default('amd64') }}"
```

Comment:
This is not how defaults works. Just put it like this:
```
# Arch of Docker images and needed packages
image_arch: amd64
```
There's no need to quote if it starts with a letter.